src/tutorials/konnector/index.md

```diff
@@ -5,6 +5,12 @@ To protect your data, each connector runs inside a container in order to sandbox
 
 > ⚠️ For historical reasons, in the Cozy codebase, a cozy connector is named konnector, please follow this convention if modifying an existing application.
 
+What will you need to start ?
+
+- Node 16, you can check your version with `node --version` in your shell.
```

Comment:
Not sure it's relevant to say how to check the node version. Maybe you can add a link to nodejs website